modules/timedrift/timedrift.go

```diff
@@ -106,12 +118,31 @@ func (r *run) Run(in io.Reader) (out string) {
 	}
 	// assume host has synched time and set to false if not true
 	el.IsWithinDrift = true
+
+	//Load ntp servers from /etc/ntp.conf
+	ntpFile, err = os.Open("/etc/ntp.conf")
+	if err != nil {
+		panic(err)
```

Comment:
The module should just continue with the backup ntp hosts if `/etc/ntp.conf` is unreadable, and store it into the returned errors:
```go
r.Results.Errors = append(r.Results.Errors, fmt.Sprintf("Using backup NTP hosts. Failed to read /etc/ntp.conf with error '%v'", err))
```